WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -488,7 +488,7 @@
 /* Noun. Links to a blog's Activity screen. */
 "Activity Log" = "Activity Log";
 
-/* Option in Support view to see activity logs. */
+/* No comment provided by engineer. */
 "Activity Logs" = "Activity Logs";
```

Comment:
This string is self-explanatory, IMHO. Still, it wouldn't hurt to use the fact that the key is not a reverse DNS one to make a PR to update it and while at it restore/update the comment to give more context to the reviewers.